src/services/company-code.ts

```diff
@@ -0,0 +1,17 @@
+import { Storage } from '@capacitor/storage'
+
+const companyCodeKey = 'companyCode'
+
+export const companyCodeFetcher = async (): Promise<string | undefined> => {
```

Comment:
Not sure what is more clear - expressly stating we might return undefined, or that we try and return a string... haha -- I need more TS exposure.

```suggestion
export const companyCodeFetcher = async (): Promise<string?> => {
```

Reply:
That suggestion actually results in a compilation error. I'm not sure if there's a better way to do it.